policy/image_quality_weights/lsst_gaussian_quadrature_weights.yaml

```diff
@@ -0,0 +1,39 @@
+---
+
+# Y2 data defined in y = A*x + y2, where y2 = A2*x2.
+# This data corresponds to the offsets to each of the Gaussian Quadrature points
```

Comment:
Gaussian Quadrature -> Gaussian Quadrature (GQ), this is because you used the "GQ" abbreviation below. This modification applies to all the `yaml` files.